File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/i/ICiMrParamService.java
package iih.ci.mr.i;

import xap.mw.core.data.BizException;

public interface ICiMrParamService {

	/**
	 * 获取编辑器配置参数
	 * @return
	 * @throws BizException
	 */
	public abstract String GetEditorConfig(String id_org) throws BizException;
	
	/**
	 * 获取门诊病历保存时是否判断必填项标识
	 * @param id_org
	 * @return
	 */
	public abstract Boolean getCiMrRequiredField(String id_org);
	
	/**
	 * 获取门诊病历保存时是否判断互斥项标识
	 * @param id_org
	 * @return
	 */
	public abstract Boolean getCiMrMutexField(String id_org);
	
	/**
	 * 获取门诊病历打印时是否预览
	 * @param id_org
	 * @return
	 */
	public abstract Boolean getSysParamCiMrPrintPreView(String id_org);
}

File: client/iih_stub/iih.ci_stub/iih.ci.mr/i/ICiMrParamService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace iih.ci.mr.i
{
    public interface ICiMrParamService
    {
        /// <summary>
        /// 获取编辑器配置参数
        /// </summary>
        /// <returns></returns>
        string GetEditorConfig(string id_org);

        /// <summary>
        /// 获取门诊病历保存时是否判断必填项标识
        /// </summary>
        /// <param name="id_org"></param>
        /// <returns></returns>
        bool GetCiMrRequiredField(String id_org);

       /// <summary>
         /// 获取门诊病历保存时是否判断互斥项标识
       /// </summary>
       /// <param name="id_org"></param>
       /// <returns></returns>
         bool GetCiMrMutexField(String id_org);

        /// <summary>
        /// 获取门诊病历打印时是否预览
        /// </summary>
        /// <param name="id_org"></param>
        /// <returns></returns>
         bool GetSysParamCiMrPrintPreView(String id_org);
    }
}
